internal/service/domain/sensor/sensor.go

```diff
@@ -100,6 +106,46 @@ func (s *SensorService) Delete(ctx context.Context, id string) error {
 	return nil
 }
 
+func (s *SensorService) RunStatusUpdater(ctx context.Context, interval time.Duration) {
+	ticker := time.NewTicker(interval)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			slog.Info("EVERY two min")
+			err := s.updateStaleSensorStatuses(ctx)
+			if err != nil {
+				slog.Error("Error updating sensor statuses", "error", err.Error())
```

Comment:
This message sounds a bit odd, how about this

```suggestion
				slog.Error("Failure to update sensor status", "error", err.Error())
```